### ISSUE TITLE
Create Groups to work on questions in Vilfredo

### ISSUE THREAD

**Issue by pietrosperoni** [fairdemocracy/vilfredo-client#113]
It should be possible to define lists of people. In a way similar to Twitter or spotify: each person can define a list of people. Lists are public. Each person can copy a list (and then modify it). And then it is possible to add a whole list to a question. As long as the sum of all the participants (at least of all the voters) is not more than 25.

**Comment by giovanisp** [comment]
"Lists are public".
You mean lists are shared among everyone in the system ? What if I don't want to appear in a list called "ABC" by some other user and used by others, because I don't share the name given to it, or the other names in the list (so I'd need to remove myself everytime other user adds the list)? Specially for influent people that might be added to thousands of "public lists" ?
You did you mean lists are managed like facebook and g+ (the user can manage his groups for easier invitations - sample lists "Department A managers" - "Key employees" - "Board of Directors 2014" and so on...).

**Comment by pietrosperoni** [comment]
Well twitter lists are public. Why is that not a problem there?
I am not sure what twitter does to avoid people being added to lists with offensive names

**Comment by giovanisp** [comment]
Because twitter lists are made so you can "follow" (read) other people's tweets. In our case lists would be used to "INVITE" them to questions... which puts a burden in their email boxes. If i am added to a twitter list I know I am being read by more people. If I get added to a vilfredo public list, it means I am being "demanded" to work in a question... In this case a system like facebook lists (with permission to be added on another users' private list) would be better in my opinion.

**Comment by pietrosperoni** [comment]
You are right, so people should add people to list, and the other must agree to be added.

But who can then use a list? Only the first creator? This would be too limiting. 
But everybody would be also dangerous.

**Comment by giovanisp** [comment]
if lists are private (just like as facebook lists or google circles) we can
1 - do like facebook lists, that is, you can add other users after their consent
2 - do like google circles, that is, you can add other to the list without their consent.

all of this if lists are private.

**Comment by pietrosperoni** [comment]
But we need to be able to invite people so they can also use the list. Suppose I make a list for a project. Then will it only be me able to ask a question to this list?

What if instead a list is a group. Like facebook groups. Everybody who is inside knows it. And you can ask questions to the group. If you are outside of the group you can propose a question to the group, but it only gets asked if someone from the group takes the responsibility to ask it. Like laws in Italy can only be presented by PM, but anyone can suggest to a PM to write a law. They will just ignore you (unless you pay them... but that is another story)

**Comment by giovanisp** [comment]
+1 to work on groups, but later.

**Comment by pietrosperoni** [comment]
Thinking about it, probably we should also implement lists, but groups are important as something in and by itself. You join a group, and then you can participate in all questions inside this group.

I would say we definitely should ask the question: "how should groups be organised on Vilfredo" on vilfredo.

**Comment by pietrosperoni** [comment]
not only that, but you also can have a history of the decisions taken. and for each decision it could add to the calendar when the decisions should be taken again. At which point Vilfredo might call for a meeting

**Comment by giovanisp** [comment]
+1
Let's move all our "questions" on github to vilfredo 2.0.0-rc1 after its release on www.vilfredo.org ?

**Comment by pietrosperoni** [comment]
yes!
